### DISCUSSION TITLE
Feature Request - Modern Dashboard UI

### DISCUSSION THREAD

**Discussion by rehanone** [TechnitiumSoftware/DnsServer#652] [Ideas]
I would like to suggest new UI for the dashboard using something like https://github.com/tremorlabs/tremor

Based on what features it offers, it looks like it has all the UI components to create a very compelling UI for this project. It could even be deployed as a separate docker service in a compose file that interacts with the service using REST api.

**Comment by ShreyasZare** [comment]
Thanks for the request. This is however not feasible to be implemented just because of lack of time. The time that is available is being used to implement core DNS features and fixing issues. UI is thus not on priority and changing the entire UI will be a huge task by itself. Maybe in future once most of the core DNS features are implemented and there is time available that the UI can be worked on.
